.github/workflows/stale.yaml

```diff
@@ -0,0 +1,24 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 7 * * 1-5' # every morning at 7 AM, only work days
+
+permissions:
+  contents: write
```

Comment:
I think this is needed for delete-branch option, but we don't want the stale bot to close PRs

Reply:
You're right, I'll remove it